src/main/scala/is/hail/annotations/UnsafeUtils.scala

```diff
@@ -14,8 +14,8 @@ object UnsafeUtils {
   }
 
   def roundUpAlignment(offset: Long, alignment: Long): Long = {
-    assert(alignment > 0, s"invalid alignment: $alignment")
-    assert((alignment & (alignment - 1)) == 0, s"invalid alignment: $alignment") // power of 2
+    assert(alignment > 0)
```

Comment:
Prefer:
```scala
assert(Option(alignment).filter(_.min(0) == 0).isDefined)
```

👎

Reply:
sorry, should have been
```scala
assert(Option(alignment).map(_.min(0) == 0).map(_.toString).forall(_ == "true"))
```

Reply:
Now all it's missing is a helpful error message.